### PR TITLE
naming, new properties

### PR DESCRIPTION
**Pull request by travisghansen**
Trying to better align `_active` vs `_enabled` with upstream terminology.

Added a bunch of new properties. Still not everything but closer.

### CODE REVIEW

**Comment by travisghansen** [comment]
@PeteBa can you review this against your knowledge of OpenSprinkler and let me know if the terminology better aligns?

**Comment by travisghansen** [comment]
On the integration side I think it would just be enabled but then adding a bunch of extra datapoints to the state attributes (the one changes meaning, but it's the same name I think).

My brain is starting to go in circles with this..but I really like how the integration is coming together honestly. I think with a little more work on the services we'll have a kick-ass 1.0 :)

**Comment by vinteo** [comment]
Should we merge this?

**Comment by travisghansen** [comment]
Let me see if I can get 1 or 2 more datapoints in there (trying to figure out the externalIP bit shifting stuff). Then I'd say yes. Then we can begin to update the integration with all the new stuff.

File: pyopensprinkler/__init__.py [hunk 1]
```diff
@@ -369,6 +369,26 @@ def last_run_end_time(self):
...
 
+    @property
+    def rssi(self):
+        """Retrieve RSSI"""
+        return self._get_variable("RSSI")
+
+    @property
+    def latitude(self):
+        """Retrieve latitude"""
+        return float(self._get_variable("loc").split(",")[0].strip())
```
Comment (vinteo) [inline pyopensprinkler/__init__.py:380]: Will `loc` always be set? Might need to check for `None` or empty string

Comment (travisghansen) [reply pyopensprinkler/__init__.py:380]: K, I'll code it up.

Comment (travisghansen) [reply pyopensprinkler/__init__.py]: Done.

**Comment by vinteo** [comment]
Doing a release

**Pull request merged**